mirgecom/simutil.py

```diff
@@ -21,6 +21,13 @@
 --------------
 
 .. autofunction:: generate_and_distribute_mesh
+
+File comparison utilities
```

Comment:
@MTCam : Is this the right file for these functions?

Reply:
Good question.   I think that it will do no harm putting it here for now.  I'd also be ok with adding a new module like `solutions` or something like that for comparing solutions together.

Eventually, it would be nice to have this arranged into utils that worked like:

```python
black_file = "/path/to/black_file"
blue_file = "/path/to/blue_file"
black_soln = get_soln_from_file(black_file)
blue_file = get_soln_from_file(blue_file)
soln_errors = compare_solutions(black_soln, blue_soln)
```

That way the user (e.g., driver developer) could use these pieces to write their own comparison utilities, for example.